keyboards/kona_classic/keymaps/iso_split_arrows/keymap.c

```diff
@@ -55,27 +59,17 @@ void matrix_scan_user(void) {
 
 }
 
-bool process_record_user(uint16_t keycode, keyrecord_t *record) {
-    return true;
-}
-
 void led_set_user(uint8_t usb_led) {
 
 }
 
-enum function_id {
-    ESCAPE,
-};
+bool process_record_user(uint16_t keycode, keyrecord_t* record) {
+  static uint8_t shift_esc_shift_mask;
 
-const uint16_t PROGMEM fn_actions[] = {
-  [0]  = ACTION_FUNCTION(ESCAPE),
-};
+  switch (keycode) {
+    case SFT_ESC:
+      shift_esc_shift_mask = get_mods() & MODS_SHFT_MASK;
 
-void action_function(keyrecord_t *record, uint8_t id, uint8_t opt) {
-  static uint8_t shift_esc_shift_mask;
-  switch (id) {
-    case ESCAPE:
-      shift_esc_shift_mask = get_mods()&MODS_SHFT_MASK;
       if (record->event.pressed) {
         if (shift_esc_shift_mask) {
```

Comment:
```suggestion
        if (get_mods() & MODS_SHIFT_MASK) {
```
Same changes as above.